llvm/docs/BitCodeFormat.rst

```diff
@@ -1085,6 +1085,7 @@ The integer codes are mapped to well-known attributes as follows.
 * code 77: ``elementtype``
 * code 78: ``disable_sanitizer_instrumentation``
 * code 79: ``nosanitize_bounds``
+* code 88: ``optdebug``
```

Comment:
if you're going to update this, can you first add the attributes in between?

Reply:
This could/should be committed separately/before this change - since it's fixing an existing documentation problem, not directly related to the patch adding optdebug (eg: if we need to revert the optdebug patch for any reason, we wouldn't want to lose this work)